addons/binding/org.openhab.binding.zoneminder/README.md

```diff
@@ -49,15 +49,48 @@ When a new monitor is discovered it will appear in the Inbox.
+| urlApi                   |    X     | Path to ZoneMinder API (Default: '/zm/api')                                                |
+| portHttp                 |    X     | Port to access ZoneMinder site. (Default: 0 (is either 80 or 443 depending on protocol))   |
+| portTelnet               |    X     | Port to access ZoneMinder with Telnet (Default: 6802)                                      |
+| refreshNormal            |    X     | Refresh rate in seconds for Normal priority (Default: 10)                                  |
+| refreshLow               |    X     | Refresh rate in seconds for Low priority (Default: 60)                                     |
+| diskUsageRefresh         |    X     | Either 'batch' or 'disabled' (Default: 'disabled')                                         |
+| autodiscover             |    X     | Enable / Disable autodiscovery (Default: true)                                             |
+| useSpecificUserStreaming |    X     | Use specific user for streaming (Default: false)                                           |
+| streamingUser            |    X     | If 'useSpecificUserStreaming' is true, username must be specified here                     |
+| streamingPassword        |    X     | If 'useSpecificUserStreaming' is true, password must be specified here                     |
+
+#### Monitor Configuration
+| Parameter                | Optional | Description                                                                                   |
+|--------------------------|----------|-----------------------------------------------------------------------------------------------|
+| id                       |          | Id of the monitor. Must match id in ZoneMinder                                                |
+| triggerTimeout           |    x     | Timeout in seconds of events generated from openHAB (Default: 60)                             |
+| eventText                |    X     | Event text of openHAB trigegred events (Default: 'Triggered from openHAB')                    |
+| imageRefreshIdle         |    X     | Refresh rate of image when monitor has no active event (normal, low, disabled) (Default: low) |
+| imageRefreshEvent        |    X     | Refresh rate when active event (alarm, normal, low, disabled), (Default: alarm)               |
+| imageScale               |    X     | Size (scale) of image. Default: 100                                                           |
+
+
+### Things file
 ```
-Bridge zoneminder:server:ZoneMinderSample [ hostname="192.168.1.55", user="<USERNAME>", password="<PASSWORD>", telnet_port=6802, refresh_interval_disk_usage=1 ]
+Bridge zoneminder:server:ZoneMinderSample [ host="192.168.1.55", protocol="http", user="<USERNAME>", password="<PASSWORD>", autodiscover=false, useSpecificUserStreaming=true, streamingUser="<STREAMING-USER>", streamingPassword="<STREAMING-PASSWORD>" ]
```

Comment:
You have a good point, but acutally I believe not doing it, since I have released this to the marketplace longtime ago . And I believe that most of the users is running on the marketplace version (since it is far more stable).